interface/requires.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
```

Comment:
As we don't have any user of this yet we can drop this unless we do the split into a separate library for the interface implementation